src/ecore.js

```diff
@@ -861,6 +868,9 @@ EClass.values = {
 
         return _.isArray(subTypes) ? subTypes : [];
     },
+    hasSubTypes: function () {
```

Comment:
I would not add this attribute here, one reason is that it is not something that is in the ecore model, the second is that it seems that its reason to be concerns only XMI de/serialization. So I would rather add a function in the xmi.js that handle that test.